docs/blog/posts/2024/06/espresso-portable-test-run.md

```diff
@@ -0,0 +1,191 @@
+---
+authors: [boegel, casparvl, satishskamath, jngrad]
+date: 2024-06-28
+slug: espresso-portable-test-run-eurohpc
+---
+
+# Portable test run of ESPResSo on EuroHPC systems via EESSI
+
+<figure markdown="span">
+![ESPResSo logo](espresso.png){width=30%}
+</figure>
+
+Since 14 June 2024, [ESPResSo](https://espressomd.org) v4.2.2 is available in the EESSI production repository `software.eessi.io`,
+optimized for the [8 CPU targets](https://www.eessi.io/docs/software_layer/cpu_targets) that are fully supported by version 2023.06 of EESSI.
+This allows running ESPResSo effortlessly on the EuroHPC systems where EESSI is already available,
+like [Vega](https://doc.vega.izum.si) and [Karolina](https://docs.it4i.cz/karolina/introduction).
+
+On 27 June 2024, an additional installation of ESPResSo v4.2.2 that is optimized for Arm A64FX processors
+was added, which enables also running ESPResSo efficiently on [Deucalion](https://docs.macc.fccn.pt/deucalion),
+even though EESSI is not available yet system-wide on Deucalion (see below for more details).
+
+With the [portable test for ESPResSo](https://www.eessi.io/docs/test-suite/available-tests/#espresso) that is available
+in the [EESSI test suite](https://www.eessi.io/docs/test-suite) we can easily evaluate the scalability of
+ESPResSo across EuroHPC systems, even if those systems have different system architectures.
+
+## Simulating Lennard-Jones fluids using ESPResSo
+
+[Lennard-Jones fluids](https://en.wikipedia.org/wiki/Lennard-Jones_potential) model interacting soft spheres with a potential that is weakly attractive at medium range and strongly repulsive at short range. Originally designed to model noble gases, this simple setup now underpins most particle-based simulations, such as ionic liquids, polymers, proteins and colloids, where strongly repulsive pairwise potentials are desirable to prevent particles from overlapping with one another. In addition, solvated systems with atomistic resolution typically have a large excess of solvent atoms compared to solute atoms, thus Lennard-Jones interactions tend to account for a large portion of the simulation time. Compared to other potentials, the Lennard-Jones interaction is inexpensive to calculate, and its limited range allows us to partition the simulation domain into arbitrarily small regions that can be distributed to many processors.
```

Comment:
```suggestion
[Lennard-Jones fluids](https://en.wikipedia.org/wiki/Lennard-Jones_potential) model interacting soft spheres with a potential that is weakly attractive at medium range and strongly repulsive at short range. Originally designed to model noble gases, this simple setup now underpins most particle-based simulations, such as ionic liquids, polymers, proteins and colloids, where strongly repulsive pairwise potentials are desirable to prevent particles from overlapping with one another. In addition, solvated systems with atomistic resolution typically have a large excess of solvent atoms compared to solute atoms, thus Lennard-Jones interactions tend to account for a large portion of the simulation time. Compared to other potentials, the Lennard-Jones interaction is inexpensive to calculate, and its limited range allows us to partition the simulation domain into arbitrarily small regions that can be distributed among many processors.
```